sdm/src/main/resources/cds/com.sap.cds/sdm/attachments.cds

```diff
@@ -0,0 +1,18 @@
+namespace sap.attachments;
+
+using {
+    cuid,
+    managed
+} from '@sap/cds/common';
+
+
+aspect MediaData           @(_is_media_data) {
+    content   : LargeBinary; // stored only for db-based services
```

Comment:
Do we need to store the large binary ? We are only transferring the content via stream operation isnt it ?

Reply:
As discussed if we are storing in drafts for sometime this is required